### ISSUE TITLE
Class QT_ROOT_LEVEL_POOL.....is implement in both .....

### ISSUE THREAD

**Issue by stephanielucia** [talmolab/sleap#2207]
# Bug description

I managed to install SLEAP on my macbook (M1 chip), I could open the GUI and label frames, but then when I ran the training it would crash at one point. So I deinstalled and reinstalled sleap s described on the webpage. However, when I try to open the GUI I get this error message: 

(base) stephanie@staff-net-lfx-2982 sleap % conda activate sleap
(sleap) stephanie@staff-net-lfx-2982 sleap % sleap-label
objc[8335]: Class QT_ROOT_LEVEL_POOL__THESE_OBJECTS_WILL_BE_RELEASED_WHEN_QAPP_GOES_OUT_OF_SCOPE is implemented in both /Users/stephanie/miniforge3/envs/sleap/lib/libQt6Core.6.9.1.dylib (0x15dab96c8) and /Users/stephanie/miniforge3/envs/sleap/lib/libQt5Core.5.15.15.dylib (0x17a6312f8). This may cause spurious casting failures and mysterious crashes. One of the duplicates must be removed or renamed.
objc[8335]: Class KeyValueObserver is implemented in both /Users/stephanie/miniforge3/envs/sleap/lib/libQt6Core.6.9.1.dylib (0x15dab96f0) and /Users/stephanie/miniforge3/envs/sleap/lib/libQt5Core.5.15.15.dylib (0x17a631320). This may cause spurious casting failures and mysterious crashes. One of the duplicates must be removed or renamed.
objc[8335]: Class RunLoopModeTracker is implemented in both /Users/stephanie/miniforge3/envs/sleap/lib/libQt6Core.6.9.1.dylib (0x15dab9790) and /Users/stephanie/miniforge3/envs/sleap/lib/libQt5Core.5.15.15.dylib (0x17a631370). This may cause spurious casting failures and mysterious crashes. One of the duplicates must be removed or renamed.
Saving config: /Users/stephanie/.sleap/1.4.1/preferences.yaml
zsh: segmentation fault  sleap-label
(sleap) stephanie@staff-net-lfx-2982 sleap % 

I tried deleting one of the duplicate files, but then it cannot find the files when I try to start the GUI. I tried to reinstall multple times now, even in different conda environments (with different names). 

I am new to this and don't have lots of knowledge when it comes to these issues.

Help is very much appreciated! 

Cheers, 
Steph 

## Your personal set up

-   OS:

-   Version(s):

-   SLEAP installation method (listed [here](https://sleap.ai/installation.html#)):
    -   [ ] [Conda from package](https://sleap.ai/installation.html#conda-package)
    -   [ ] [Conda from source](https://sleap.ai/installation.html#conda-from-source)
    -   [ ] [pip package](https://sleap.ai/installation.html#pip-package)
    -   [ ] [Apple Silicon Macs](https://sleap.ai/installation.html#apple-silicon-macs)

<details><summary>Environment packages</summary>

```
# paste output of `pip freeze` or `conda list` here
```

</details>

<details><summary>Logs</summary>

```
# paste relevant logs here, if any
```

</details>

**Comment by eberrigan** [comment]
Hi @stephanielucia,

We don't officially support Mac M1 anymore, but from your error it looks like you have two versions of QT installed. Please delete one of them.

Best,

Elizabeth

**Comment by stephanielucia** [comment]
Hi Elizabeth

Thanks for your reply and the info. But you support newer versions of the Mac M chips? Just not the M1? 

I tried that, did not work. Will try again. 

Best, 
Stephanie

**Comment by eberrigan** [comment]
Hi @stephanielucia,

Yes the dependencies for SLEAP are pretty complicated because of the tensorflow backend which uses the GPU. We have had to maintain multiple dependency lists depending on OS and hardware. Since the Mac chips are evolving so quickly, we had to stop supporting the M1 chips to prioritize the newer chips. As of now, we are having dependency issues with M1 and M4 using our standard installation, so users will have to troubleshoot these independently. #2212 

We are hoping once we update our entire backend to use pytorch the dependencies will be more manageable and we can extend support.

Best of luck and maybe try the discussions as well since others should be troubleshooting also.

Thanks,

Elizabeth

**Comment by talmo** [comment]
Hi @stephanielucia — apologies for the long silence. As Elizabeth noted, this is the same conda-Qt5/Qt6 collision documented in #2212. It's a legacy-1.4.1 environment issue specific to the old conda install path on Apple Silicon.

The clean fix is to upgrade to v1.5+, which switched to PySide6 (so only one Qt is in play) and migrated to PyTorch with native MPS support on Apple Silicon. Install with `uv` (no conda channel roulette) — [installation guide](https://docs.sleap.ai/latest/installation/).

If you absolutely need to stay on v1.4.1, the community workaround is to constrain the env to Qt5 by adding `"qt<6"` to your env yaml (originally posted in neuroinformatics-unit/movement#635).

Closing as completed — please open a new issue if anything's still broken on v1.5+.
